app/root.css

```diff
@@ -117,6 +117,13 @@ h2 {
 
 /* typography classes */
 
+.extra-large {
```

Comment:
<img width="273" alt="image" src="https://github.com/user-attachments/assets/bf6798fe-68e4-4369-89fa-ad225f3d06ac">
Ah, I see what you were going for. This was my mistake. I don't want to add another extra-large class, but rather make all the large classes consistent. I think I will come in and fix this soon so I don't get confused when this system doesn't match my Figma later. For now it works, sorry for the confusion.